guides/Building Your Bot/CreatingPointsSystems.md

```diff
@@ -43,7 +43,7 @@ module.exports = class extends Monitor {
 
 Some social bots have level up messages. How do we set it up? There are two ways to achieve this:
 
-1. We calculate the current level and the next level on the fly. This system is, however, harder to implement and it processes a lot of maths, but it's also RAM friendly for massive bots. We won't cover this in the guide.
+1. We calculate the current level and the next level on the fly. This system is, however, harder to implement and it requires more arithmetic calculations, but it's also RAM friendly for massive bots. We won't cover this in the guide.
```

Comment:
This sentence is really weird. Somewhat run-on, and however at the not-quite-beginning makes it clumsy feeling.